hmp/transform.py

```diff
@@ -274,13 +309,14 @@ def __init__(
             data.attrs["mcca_weights"] = mcca_m.mcca_weights
             data.attrs["pca_weights"] = mcca_m.pca_weights
 
-        elif method is None:
+        elif method is AnalysisMethod.NO_ANALYSIS:
 
             data = data.rename({"channels": "component"})
             data["component"] = np.arange(len(data.component))
             data.attrs["pca_weights"] = np.identity(len(data.component))
 
-        if apply_zscore:
+
+        if apply_zscore.apply():
```

Comment:
You could use a `__bool__`  dunder to make it easier to read in this context I think. (I.e. you just need to rename the apply method to `__bool__`.)